test/worker_test.py

```diff
@@ -838,5 +839,53 @@ def test_purge_hung_worker_override_timeout_time(self, mock_time):
         self.assertEqual(0, len(w._running_tasks))
 
 
+class ForkBombTask(luigi.Task):
+    depth = luigi.IntParameter()
+    breadth = luigi.IntParameter()
+    p = luigi.Parameter(default=(0, ))  # ehm for some weird reason [0] becomes a tuple...?
+
+    def output(self):
+        return MockFile('.'.join(map(str, self.p)))
+
+    def run(self):
+        with self.output().open('w') as f:
+            f.write('Done!')
+
+    def requires(self):
+        if len(self.p) < self.depth:
+            for i in xrange(self.breadth):
```

Comment:
As my previous comment, can you change this to range or six.moves.range(self.breadth). Otherwise, the test won't pass under python3.